.rtd-environment.yml

```diff
@@ -5,34 +5,31 @@ dependencies:
   - python
   - pip
   - openjpeg
-  - numpy
```

Comment:
Moved these packages to pip install since we actually test this config on our ci so we don' have to deal with any potential conda package issues with RTD.